src/store/modules/wallet.js

```diff
@@ -869,15 +850,17 @@ const actions = {
   },
   async walletFetchSessionUserData(
     { dispatch },
-    { shouldSkipUserInfo = false } = {}
+    { awaitForWalletEvents = false } = {}
```

Comment:
where does this use?

Reply:
Is it not used yet, but setting this flag would mimic the original behaviour, which might be useful else where